integration/simulation/simulation_test.go

```diff
@@ -53,38 +98,36 @@ func checkBlockchainValidity(t *testing.T, l1Network L1NetworkCfg, l2Network L2N
 const L1EfficiencyThreashold = 0.2
 const L2EfficiencyThreashold = 0.3
 
-// Sanity check
-func validateL1(t *testing.T, b *common.Block, s *Stats, db enclave2.DB) {
+// validateL1 does a sanity check on the mock implementation of the L1
+func validateL1(t *testing.T, stats *Stats, l1Height int, l1HeightHash common.L1RootHash, node *ethereum_mock.Node) {
```

Comment:
I'd say it's a bit cleaner to just pass in one "block resolver"

Reply:
Reduced a bit the arguments used. 👍